libraries/chef_pyenv_script_helpers.rb

```diff
@@ -1,29 +1,17 @@
-require 'chef/mixin/shell_out'
-
 class Chef
   module Pyenv
     module ScriptHelpers
+      require 'chef/mixin/shell_out'
```

Comment:
Not seeing a benefit to moving this inside the module definition as it'll always be already loaded by Chef by the time the module is `require`'d

Reply:
Not a huge benefit really. It's just bundling it with the module, in a.. Hey this is using it.